Code/Engine/Core/ResourceManager/Implementation/ResourceHandleReflection.h

```diff
@@ -0,0 +1,152 @@
+#pragma once
+
+#include <Core/CoreDLL.h>
+#include <Core/ResourceManager/ResourceManager.h>
+
+/// \brief Adds two member functions to a class, GetXyzFile() and SetXyzFile() with Xyz being equal to 'name', which allow to access the handle through strings.
```

Comment:
Had to move the new stuff into a separate file, to include it after ResourceManager.h because of that dependency.